src/EditorFeatures/Core/Implementation/NavigateTo/NavigateToItemProvider.Searcher.cs

```diff
@@ -92,8 +124,6 @@ private void Search(IDisposable navigateToSearch, IAsyncToken asyncToken)
                         }
                     }
                 }
-
-                _progress.ItemCompleted();
```

Comment:
we still have it.  it was just pulled up into the calling method.  It means that this method can return at any time without it being a problem for progress.